### PR TITLE
added method to expose gravity control.

### PR DESCRIPTION
**Pull request by akashraghav**
Hey @DanielMartinus 

added method to expose vertical/bottom gravity control. I think this would be helpful in controlling the momentum of the confettis with higher speeds, so that they don't just jump out of screen, when burst in upward direction. Kept the defaults same.

I'm also thinking to work on gravity control on x-axis, to allow change in gravity direction as well. let me know what you think.

### CODE REVIEW

**Comment by DanielMartinus** [comment]
@akashraghav thank you for your contribution. Could you run `./gradlew spotlessApply` to make sure the CI succeeds?

File: konfetti/src/main/java/nl/dionsegijn/konfetti/ParticleSystem.kt [hunk 1]
```diff
@@ -147,6 +148,15 @@ class ParticleSystem(private val konfettiView: KonfettiView) {
         return this
     }
 
+    /**
+     * Set bottom gravity
+     * If the value is negative it will be set to default value.
+     */
+    fun setGravity(y: Float): ParticleSystem {
+        gravity.y = if (y < 0) 0.01f else y
```
Comment (DanielMartinus) [inline konfetti/src/main/java/nl/dionsegijn/konfetti/ParticleSystem.kt:156]: Minor suggestion:

```Kotlin
gravity.y = y.coerceAtLeast(0.01f)
```

**Review by DanielMartinus — CHANGES_REQUESTED** [review]

File: konfetti/src/main/java/nl/dionsegijn/konfetti/ParticleSystem.kt (hunk 1 above)
Comment (akashraghav) [reply konfetti/src/main/java/nl/dionsegijn/konfetti/ParticleSystem.kt]: Updated code as suggested.

Comment (akashraghav) [reply konfetti/src/main/java/nl/dionsegijn/konfetti/ParticleSystem.kt]: > @akashraghav thank you for your contribution. Could you run `./gradlew spotlessApply` to make sure the CI succeeds?

I did this earlier and pushed the change reported in log earlier, you can view diff `konfetti/src/main/java/nl/dionsegijn/konfetti/Confetti.kt` but the CI check is still reporting same issue. While, I'm getting build success locally on both `./gradlew spotlessCheck` and `./gradlew spotlessApply` calls. Is there a way to trigger check again?

```
./gradlew spotlessCheck

> Configure project :demo-google-play
Warning: The 'kotlin-android-extensions' Gradle plugin is deprecated. Please use this migration guide (https://goo.gle/kotlin-android-extensions-deprecation) to start working with View Binding (https://developer.android.com/topic/libraries/view-binding) and the 'kotlin-parcelize' plugin.

BUILD SUCCESSFUL in 4s
5 actionable tasks: 5 executed

```

```
./gradlew spotlessApply

> Configure project :demo-google-play
Warning: The 'kotlin-android-extensions' Gradle plugin is deprecated. Please use this migration guide (https://goo.gle/kotlin-android-extensions-deprecation) to start working with View Binding (https://developer.android.com/topic/libraries/view-binding) and the 'kotlin-parcelize' plugin.

BUILD SUCCESSFUL in 926ms
5 actionable tasks: 1 executed, 4 up-to-date
```

**Comment by akashraghav** [comment]
> @akashraghav thank you for your contribution. Could you run `./gradlew spotlessApply` to make sure the CI succeeds?

I did this earlier and pushed the change reported in log, you can view diff `konfetti/src/main/java/nl/dionsegijn/konfetti/Confetti.kt` but the CI check is still reporting same issue. While, I'm getting build success locally on both `./gradlew spotlessCheck` and `./gradlew spotlessApply` calls. Is there a way to trigger check again?

```
./gradlew spotlessCheck

> Configure project :demo-google-play
Warning: The 'kotlin-android-extensions' Gradle plugin is deprecated. Please use this migration guide (https://goo.gle/kotlin-android-extensions-deprecation) to start working with View Binding (https://developer.android.com/topic/libraries/view-binding) and the 'kotlin-parcelize' plugin.

BUILD SUCCESSFUL in 4s
5 actionable tasks: 5 executed

```

```
./gradlew spotlessApply

> Configure project :demo-google-play
Warning: The 'kotlin-android-extensions' Gradle plugin is deprecated. Please use this migration guide (https://goo.gle/kotlin-android-extensions-deprecation) to start working with View Binding (https://developer.android.com/topic/libraries/view-binding) and the 'kotlin-parcelize' plugin.

BUILD SUCCESSFUL in 926ms
5 actionable tasks: 1 executed, 4 up-to-date
```

**Comment by DanielMartinus** [comment]
You're right, something is wrong with Spotless. I got it fixed locally but the CI keeps throwing the same error even though the formatting is correct. I'll let you know once its fixed

**Comment by DanielMartinus** [comment]
Figured it out. The CI is running the workflow in `pull_request_target` mode. I had to configure the workflow like that so it would also run for forked pull request like this. However, unfortunately it will take the base branch (master) and run spotless on that branch. I'll have to find a fix for that but for now it's fixed if you merge master into your PR.

**Comment by akashraghav** [comment]
Seems, I did something wrong, I'll close this and create a new PR.

**Comment by DanielMartinus** [comment]
@akashraghav I fixed it for you

**Review by DanielMartinus — APPROVED** [review]

**Comment by DanielMartinus** [comment]
Thanks for your contribution @akashraghav

**Pull request merged**